Frontend/src/Components/ListingDetails/ListingDetails.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useState } from "react";
```

Comment:
if this is a completely new file i would put it in a separate commit in the future!

Reply:
i do remember seeing it in a previous commit though so i'm not sure why it's saying it's a new file. but yeah just a reminder for the future anyways

Reply:
alright, noted